internal/elasticsearch/ingest/datastream.go

```diff
@@ -20,37 +19,20 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/elastic/elastic-package/internal/elasticsearch"
-	"github.com/elastic/elastic-package/internal/elasticsearch/ingest"
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
 var ingestPipelineTag = regexp.MustCompile(`{{\s*IngestPipeline.+}}`)
 
-type simulatePipelineRequest struct {
-	Docs []pipelineDocument `json:"docs"`
-}
-
-type pipelineDocument struct {
-	Source json.RawMessage `json:"_source"`
-}
-
-type simulatePipelineResponse struct {
-	Docs []pipelineIngestedDocument `json:"docs"`
-}
-
-type pipelineIngestedDocument struct {
-	Doc pipelineDocument `json:"doc"`
-}
-
-func installIngestPipelines(api *elasticsearch.API, dataStreamPath string) (string, []ingest.Pipeline, error) {
+func InstallDataStreamPipelines(api *elasticsearch.API, dataStreamPath string) (string, []Pipeline, error) {
```

Comment:
nit: is this renaming intentional? AFAIR there are packages with ingest pipelines detached from data streams, but most likely we don't support them here.

Reply:
Both signatures received datastream as a parameter, since I had to expose it, it seemed clearer to me to rename it.